.circleci/config.yml

```diff
@@ -320,3 +332,4 @@ workflows:
       - build-nightly-buster-securedrop-log:
           requires:
             - build-nightly-buster-securedrop-export
+      - build-nightly-buster-securedrop-workstation-svs-disp
```

Comment:
let's have this `require` `build-nightly-buster-securedrop-log` (this is so the jobs don't race each other, see comment above the `nightly` key)

this will conflict with #129 but we can rebase whichever gets merged last